sdk/spring/spring-cloud-azure-feature-management/README.md

```diff
@@ -192,10 +192,136 @@ feature-management:
         -
          name: TimeWindowFilter
           parameters:
-            time-window-filter-setting-start: "Wed, 01 May 2019 13:59:59 GMT",
-            time-window-filter-setting-end: "Mon, 01 July 2019 00:00:00 GMT"
+            Start: "Wed, 01 May 2019 13:59:59 GMT",
+            End: "Mon, 01 July 2019 00:00:00 GMT"
 ```
 
+The time window can be configured to recur periodically. This can be useful for the scenarios where one may need to turn on a feature during a low or high traffic period of a day or certain days of a week. To expand the individual time window to recurring time windows, the recurrence rule should be specified in the `Recurrence` parameter.
+
+**Note:** `Start` and `End` must be both specified to enable `Recurrence`.
+
+```yaml
+feature-management:
+  feature-flags:
+    feature-v:
+      enabled-for:
+        -
+         name: TimeWindowFilter
+          parameters:
+            start: "Fri, 22 Mar 2024 20:00:00 GMT",
+            end: "Mon, 01 July 2019 00:00:00 GMT",
+            recurrence:
+              pattern:
+                type: "Daily",
+                interval: 1
+              range:
+                type: "NoEnd"
```

Comment:
When using the recurrence pattern, we need to provide both `start` and `end`. The time window is `end` - `start`. 
If the range type is "NoEnd", then we don't need to provide other paramters(`endDate`, `numberOfOccurrences`) in range scope.
For example, if we want to enable a feature flag in 2:00 AM ~3:00 AM every day from 2024/05/11. Then the config yaml should be:
```
start: "Sat, 11 May 2024 02:00:00 GMT",
end: "Sat, 11 May 2024 03:00:00 GMT",
recurrence:
  pattern:
    type: "Daily",
    interval: 1
  range:
    type: "NoEnd"
```

If we want to enable a feature flag in 2:00 AM ~3:00 AM every day from 2024/05/11 to 2024/06/11. Then the config yaml should be:
```
start: "Sat, 11 May 2024 02:00:00 GMT",
end: "Sat, 11 May 2024 03:00:00 GMT",
recurrence:
  pattern:
    type: "Daily",
    interval: 1
  range:
    type: "EndDate",
    endDate: "Tue, 11 June 2024 00:00:00 GMT"
```

Reply:
I think the intro paragraph no longer works as it goes agains the recurring items. Maybe instead of what we have here where both are one section we make it 2 different sections. I know it's the same filter, but they don't operate the same.

Reply:
But if we make it into 2 difference sections, then it's also confused. It looks like we have 2 fiters with same name "TimeWindowFilter", different parameters. 
Another solution, how about updating the example, not using `NoEnd`, use `Numbered` instaed?
```
start: "Fri, 22 Mar 2024 20:00:00 GMT",
end: "Sat, 23 Mar 2024 02:00:00 GMT",
recurrence:
  pattern:
    type: "Daily",
    interval: 1
  range:
    type: "Numbered"
    numberOfOccurrences: 3
```